### PR TITLE
A

### PR DESCRIPTION
**Pull request by deiwin**

### CODE REVIEW

File: a [hunk 1]
```diff
@@ -0,0 +1 @@
+a
```
Comment (deiwin) [inline a]: this should be `b`

**Comment by deiwin** [comment]
!merge

**Comment by deiwin** [comment]
!squash

**Comment by deiwin** [comment]
!check

**Comment by indrekj** [comment]
I thought @deiwin went mad when read the email notification

**Comment by deiwin** [comment]
I was trying to show off the bot ... And then it failed.

**Comment by deiwin** [comment]
```
2017/03/23 00:29:11 Checking for fixup commits for PR salemove/github-review-helper#32.
2017/03/23 00:29:11 Getting commits for PR salemove/github-review-helper#32 failed. Received an unexpected head with SHA of b1d3358030c47735c252ad91bf036a46d9ab5fb3.. Trying again.
2017/03/23 00:29:11 Getting commits for PR salemove/github-review-helper#32 failed. Received an unexpected head with SHA of b1d3358030c47735c252ad91bf036a46d9ab5fb3.. Trying again.
2017/03/23 00:29:11 Getting commits for PR salemove/github-review-helper#32 failed. Received an unexpected head with SHA of b1d3358030c47735c252ad91bf036a46d9ab5fb3.. Trying again.
2017/03/23 00:29:11 Responding to the HTTP request with:
2017/03/23 00:29:11 Error: Getting commits for PR salemove/github-review-helper#32 failed. Received an unexpected head with SHA of b1d3358030c47735c252ad91bf036a46d9ab5fb3.
```

So the same old race caused it to fail. I'll have to add a delay between the retries.

**Pull request merged**